src/dmd/toobj.d

```diff
@@ -652,6 +652,9 @@ void toObjFile(Dsymbol ds, bool multiobj)
                 dt2common(&s.Sdt);
             }
 
+            if (s.Sclass & SCglobal && s.Stype.Tty & mTYconst)
```

Comment:
Looks like a change from https://github.com/dlang/dmd/pull/14047 made it in here too